.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for Go modules
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
I think for the SDK we want run this `daily` and have `weekly` on the device services. This way it picks up on our go-mod changes quicker.